CHANGELOG.rst

```diff
@@ -3,7 +3,7 @@ Changelog
 
 .. _section-1:
 
-0.3.0
+0.3.1
```

Comment:
No, you need to add a new heading *only* for your own changes. The other changes need to stay under the 0.3.0 heading.